packages/block-editor/src/components/link-control/search-input.js

```diff
@@ -43,6 +42,7 @@ const LinkControlSearchInput = forwardRef(
 			withURLSuggestion = true,
 			createSuggestionButtonText,
 			hideLabelFromVision = false,
+			suffix = null,
```

Comment:
Same as above — no need to default this prop to `null`, leaving it `undefined` should just work

Reply:
Okay.